src/renderer/views/Watch/Watch.js

```diff
@@ -722,7 +722,8 @@ export default defineComponent({
           }
 
           if (result.storyboards?.type === 'PlayerStoryboardSpec') {
-            let source = result.storyboards.boards
+            /** @type {import('youtubei.js/dist/src/parser/classes/PlayerStoryboardSpec').StoryboardData[]} */
+            let source = result.storyboardSpec.boards
```

Comment:
@ChunkyProgrammer 
This blows up
![image](https://github.com/user-attachments/assets/caa2703c-6f16-4e5d-a49a-1115cbfb1089)

Reply:
Whoops, didnt mean to change this from storyboards to storyboardSpec